src/ui/common/src/components/integrations/StorageConfiguration.tsx

```diff
@@ -0,0 +1,110 @@
+import { faDatabase, faTags } from '@fortawesome/free-solid-svg-icons';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Box, Typography } from '@mui/material';
+import React, { useEffect } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+
+import { handleGetServerConfig } from '../../handlers/getServerConfig';
+import { RootState } from '../../stores/store';
+import useUser from '../hooks/useUser';
+
+interface StorageConfigDisplayProps {
+  integrationName: string;
+}
+
+export const StorageConfigurationDisplay: React.FC<
+  StorageConfigDisplayProps
+> = ({ integrationName }) => {
+  const { user } = useUser();
+  const dispatch = useDispatch();
+  const serverConfig = useSelector(
+    (state: RootState) => state.serverConfigReducer
+  );
+
+  const storageConfig = serverConfig?.config?.storageConfig;
+
+  useEffect(() => {
+    async function fetchServerConfig() {
+      if (user) {
+        await dispatch(handleGetServerConfig({ apiKey: user.apiKey }));
+      }
+    }
+
+    fetchServerConfig();
+  }, [user]);
+
+  let dataStorageInfo,
+    dataStorageText = null;
+
+  if (storageConfig) {
+    switch (integrationName) {
+      case 'file': {
+        // file is same as sqlite?
```

Comment:
SQLite is something we use just for our database. File, S3, and GCS are our options for the "metadata" storage layer, so these should be treated separately.